lib/Db/MailAccount.php

```diff
@@ -123,6 +126,8 @@ public function __construct(array $params=[]) {
 		if (isset($params['smtpPassword'])) {
 			$this->setOutboundPassword($params['smtpPassword']);
 		}
+
+		$this->addType('lastMailboxSync', 'integer');
```

Comment:
Is there a reason you only set it for this one explicitly?

Reply:
Come tot hink of it we should make this use the declarations docs properly I guess (but for server fixing that is)

Reply:
It's the only one I touch in this PR. The others are all strings and so should just with the default.